app/containers/message/Reply.tsx

```diff
@@ -115,22 +116,22 @@ interface IMessageTitle {
 
 interface IMessageDescription {
 	attachment: Partial<IMessageReplyAttachment>;
-	getCustomEmoji: Function;
+	getCustomEmoji: TGetCustomEmoji;
```

Comment:
I know you only touched this part, but do you realize that there are two interfaces with exactly the same properties? Besides the fact that they are very similar?

Maybe I could clean up this code, or just check a TODO.

Reply:
You are correct! However, I didn't "refactor" this, because I understood that they are 2 different props for 2 different components

So, WDYT? No problem for me, up to you

Reply:
Okay, as it's not necessarily your implementation, we leave it that way, but I think that at most it would create an interface that extends the other, since the data is virtually equal.